### PR TITLE
fix search display bug

### PR DESCRIPTION
**Pull request by luwcs**

### CODE REVIEW

File: wbdv-final-movie-seeker/src/components/movie-search/search-page.js [hunk 1]
```diff
@@ -40,9 +40,9 @@ const SearchPage = () => {
         {
           results && results.Search && results.Search.map((movie) => {
             return (
-              <li className="list-group-item">
+              <li key={movie.imdbID} className="list-group-item">
```
Comment (shaoxcs) [inline wbdv-final-movie-seeker/src/components/movie-search/search-page.js:43]: why you need this?

Comment (luwcs) [reply wbdv-final-movie-seeker/src/components/movie-search/search-page.js:43]: From professor's code

Comment (shaoxcs) [reply wbdv-final-movie-seeker/src/components/movie-search/search-page.js:43]: sounds good

**Review by shaoxcs — APPROVED** [review]

**Review by shaoxcs — APPROVED** [review]
LGTM!

**Pull request merged**